cEP-0010.md

```diff
@@ -140,16 +152,24 @@ Here is the list of aspects to be implemented along with their corresponding
   in your files given `Root.Formatting.Quotation.preferred_quotation`.
 - `Root.Formatting.Quotation.preferred_quotation`: This taste represents the
   user's preferred quotation.
```

Comment:
why does this say "user's preferred quotation".

a coafile is not about user preferences; instead they are about project preferences.